manifest.toml

```diff
@@ -0,0 +1,176 @@
+demo = "https://www.odoo.com/trial"
+admindoc = "https://www.odoo.com/documentation/16.0/administration.html"
+userdoc = "https://www.odoo.com/documentation/16.0/applications.html"
+code = "https://github.com/odoo/odoo"
+cpe = "cpe:2.3:a:odoo:odoo"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = "all"
+multi_instance = true
+ldap = true
+sso = true
+disk = "50M" # FIXME: replace with an **estimate** minimum disk requirement. e.g. 20M, 400M, 1G, ...
+ram.build = "50M" # FIXME: replace with an **estimate** minimum ram requirement. e.g. 50M, 400M, 1G, ...
+ram.runtime = "50M" # FIXME: replace with an **estimate** minimum ram requirement. e.g. 50M, 400M, 1G, ...
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.init_main_permission]
+    type = "group"
+    default = "visitors"
+
+    [install.app_version]
+    ask.en = "Choose the version you want"
+    ask.fr = "Choisissez la version désirée"
+    type = "string"
+    choices = ["11", "12", "14", "15", "16"]
+    default = "16"
+
+    [install.oca]
+    ask.en = "Do you want to setup OCA instead of LibreERP?"
+    ask.fr = "Désirez vous installer OCA à la place des sources de LibreERP"
+    type = "boolean"
+    default = false
+
+    [install.lang]
+    ask.en = "Choose the application language"
+    ask.fr = "Choisissez la langue de l'application"
+    type = "select"
+    choices = ["sq_AL", "am_ET", "ar_SY", "eu_ES", "bs_BA", "bg_BG", "my_MM", "ca_ES", "zh_CN", "zh_HK", "zh_TW", "hr_HR", "cs_CZ", "da_DK", "nl_BE", "nl_NL", "en_AU", "en_GB", "en_US", "et_EE", "fi_FI", "fr_BE", "fr_CA", "fr_CH", "fr_FR", "gl_ES", "ka_GE", "de_CH", "de_DE", "el_GR", "gu_IN", "he_IL", "hi_IN", "hu_HU", "id_ID", "it_IT", "ja_JP", "kab_DZ", "ko_KP", "ko_KR", "lo_LA", "lv_LV", "lt_LT", "mk_MK", "mn_MN", "nb_NO", "fa_IR", "pl_PL", "pt_BR", "pt_PT", "ro_RO", "ru_RU", "sr_RS", "sr@latin", "sk_SK", "sl_SI", "es_AR", "es_BO", "es_CL", "es_CO", "es_CR", "es_DO", "es_EC", "es_GT", "es_MX", "es_PA", "es_PE", "es_PY", "es_UY", "es_VE", "es_ES", "sv_SE", "te_IN", "th_TH", "tr_TR", "uk_UA", "vi_VN"]
+    default = "en_US"
+
+    [install.tz]
+    ask.en = "Choose a timezone"
+    ask.fr = "Choisissez un fuseau horaire"
+    type = "string"
+    default = "Etc/UTC"
+
+    [install.admin]
+    type = "user"
+
+    [install.admin_password]
+    ask.en = "Choose a strong master admin password"
+    ask.fr = "Choisissez un mot de passe maître fort pour l'administration"
+    type = "password"
+
+[resources]
+    [resources.sources]
+    [resources.sources.main_16_oca]
+    url = "https://github.com/OCA/OCB/archive/9d5ff52da5ec65a21a00a46e21783a2c6188dd80.tar.gz"
+    sha256 = "3c3a95ec627e3807c85b0b2fd13a484b0b990426efb4905681aaee8768386e39"
+    prefetch = false
+
+    [resources.sources.main_16]
+    url = "https://github.com/odoo/odoo/archive/2a88076c1d1df4920bfd139c94862a894d8ef3bc.tar.gz"
+    sha256 = "39d23f01d9a095d75460829acfd53b74bcfc273a074d6eae163eedcaeeeb58c7"
+    prefetch = false
```

Comment:
Randomly passing but it sounds like a big meh to be able to choose between the twos ... The OCA/OCB one should be equivalent but with more bugfixes etc. I would just install OCA/OCB and voila...

Reply:
Fine with me, i don't understand what OCA builds are :D

Reply:
Done.